woocommerce/woocommerce-android
# Add OrderList performance transaction

Closes: #


### Description

This PR adds performance tracking for the loading list of orders.

### Testing instructions

1. Open the app
2. Go to list of orders
3. Verify that after a few seconds, you can see a new transaction (with HTTP, db and UI spans) in Sentry Performance Monitoring dashboard. Transaction name: `OrderList` (don't forget to sort transactions as "recent first")

### Images/gif

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/5845095/182850416-d985f1cc-f80e-4c45-9d86-3ed9ff6f31eb.png">
